agent/xds/rbac.go

```diff
@@ -576,7 +632,7 @@ func makeRBACRules(
 	for i, rbacIxn := range rbacIxns {
 		var infos []*JWTInfo
 		if isHTTP {
-			infos = collectJWTInfos(rbacIxn)
+			infos = collectTopLevelJWTInfos(rbacIxn)
```

Comment:
Do we need to call this method anymore? It looks like it is just copying the list of jwtInfos from rbacIxn. Can we just do `infos = rbacIxn.jwtInfos` here?